docs/iris/src/whitepapers/change_management.rst

```diff
@@ -0,0 +1,393 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The API is the components of the iris module and submodules which are
+"public" :  In Python, by convention, this means everything whose name does not
+have a leading underscore "_".
+This means all public modules and submodules, and their contained classes,
+public data and properties, functions and methods.
+
+A "feature" of the API includes public objects as above, but may also be used
+more loosely to indicate a class or mode of behaviour, for example when a
+keyword has a specific value, like "interpolate(mode='linear')".
+
+A "usage" is any code referring to public API elements, for example :
+
+        * `print iris.thing`
+        * `iris.submodule.call(arg1)`
+        * `iris.module.call(arg1, arg2, *more_args)`
+        * `iris.module.call(arg1, arg2, control=3)`
+        * `x = iris.module.class(arg, key=4)`
+
+A "behaviour" is whatever happens when you invoke a particular API usage,
+encompassing both returned values and any side effects such as code state
+changes or data written to files.
+
+The above examples are all public feature usages, and therefore should
+continue to work, with the same behaviours, at least until the next **major**
+version release.
+
+
+
+.. _iris_change_releases:
+
+Releases and Versions
+---------------------
+
+
+Iris releases have a unique identifying version string, in the form
+"<major>.<minor>.<micro><extension>", available to code as
+:data:`iris.__version__` .
+
+This contains major and minor release numbers.  The numbering and meaning of
+these are defined, following the `SemVer project <http://semver.org/>`_.
+
+The essential aspects of the "<major>.<minor>.<micro><extension>" arrangement
+are :
+
+    * "<major>", "<minor>" and "<micro>" are all integers, thus version
+      2.12 is later than 2.2 (i.e. it is "two point twelve", not "two point one
+      two").
+
+    * "<major>.<minor>" denote the software release version.
+
+    * A non-zero "<micro>" denotes a bugfix version, thus a release "X.Y.0" may
+      be followed by "X.Y.1", "X.Y.2" etc, which *only* differ by containing
+      bugfixes.  Any bugfix release supercedes its predecessors, and does not
+      change any (valid) APIs or behaviour :  there should be no reason not to
+      replace a given version with its latest bugfix successor.
+
+    * "<extension>" is blank for formal releases.  It used to indicate
+      provisional software for testing :  The version of general Iris code in
+      development is labelled "-DEV", and release candidates for testing during
+      the release process are labelled "-rc1", "-rc2" etc.
+      For development code, the version number is that of the *next* release,
+      which this code version is progressing towards, e.g. "1.2-DEV" for code
+      following the 1.1 release and eventually giving rise to "1.2".
+
+.. note::
+    Our use of "-<extension>" is typical, but does not follow strict SemVer
+    principles.
```

Comment:
Our current incompatibility with SemVer is limited to defining versions such as "1.9.0rc1",
instead of "1.0.-rc.1" : adding a hyphen and a dot.
See the tags ...
We did at one point have "1.7.1-rc.1" thru "1.7.3-rc.1", but then reverted to "1.7.4-rc1".
Since then, we have dropped both the hyphen and the dot.
The current Developer guide also specifically requires this form, which is **PEP440 but not SemVer**.

The big problem is that PEP440 and SemVer are **mutually incompatible** in this respect :frowning: 

@SemVer 
> "A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version"
  
@PEP440 
> "[N!]N(.N)*[{a|b|rc}N][.postN][.devN]"

**IMHO** we should just change this + have done with it.